src/test/java/au/id/deejay/webserver/server/WebServerExecutorTest.java

```diff
@@ -12,24 +13,25 @@
 /**
  * @author David Jessup
  */
+@State(Scope.Benchmark)
 public class WebServerExecutorTest {
 
 	private WebServerExecutor executor;
 
-	@After
+	@TearDown(Level.Invocation)
 	public void tearDown() throws Exception {
 		if (executor != null && executor.running()) {
 			executor.stop();
 		}
 	}
 
-	@Test
+	@Benchmark
 	public void testConstructor() throws Exception {
 		executor = new WebServerExecutor(0, 10, 10, mock(ResponseFactory.class));
 		assertThat(executor.running(), is(false));
 	}
 
-	@Test
+	@Benchmark
```

Comment:
You're gonna need arguments here. What do the arguments to the `WebServerExecutor()` ctor below mean?

Reply:
https://github.com/djessup/java-webserver/blob/975ed887b8044edb81cd9e5a294c2ff590ec6256/src/main/java/au/id/deejay/webserver/server/WebServerExecutor.java#L40-L47

Reply:
Oh, so I don't think we'll need any of these ,right? We won't need max threads anymore, correct?

Reply:
Right, because newVirtualThreadPerTaskExecutor() doesn't take a thread amount as a parameter

Reply:
So, for this test, how would we increase the number of transactions being processed?